spec/system/admin/products_v3/create_spec.rb

```diff
@@ -45,6 +45,18 @@
       expect(page).to have_content "New variant"
     end
 
+    it "has the 1 unit value for the new variant display_as by default" do
+      new_variant_button.click
+
+      within new_variant_row do
+        unit_button = find('button[aria-label="Unit"]')
+        expect(unit_button.text.strip).to eq('1kg')
+
+        unit_button.click
+        find('input[aria-label="Display unit as"][placeholder="1kg"]')
```

Comment:
Shouldn't this be an expectation? Eg:
```suggestion
        expect(page).to have_field "Display unit as", placeholder: "1kg"
```